### ISSUE TITLE
Editor: Captions on image block showing white by default

### ISSUE THREAD

**Issue by jackiejade** [Automattic/wp-calypso#53422]
### Steps to reproduce the behavior

1. Go to Pages > Add New
2. Click on + to add an image block and select an image
3. Try to add a caption to the image
4. You can type below the image, but there is no box to tell you you can, and when you type the font shows up in white by default. 
5. Change text color using the options:
6. https://d.pr/i/E8yK3T 

#### What I expected to happen
It should be black or a visible color and there used to be a box around the caption so we knew there was a place to type.

#### What actually happened
No caption box and the text is white

#### Is this specific to the applied theme? Which one?
This was seen in older themes, like Intergalactic, Pilcrow and Twenty Thirteen. 

#### Does this happen on simple or atomic sites or both?
So far only simple sites.

#### Is there any console output or error text?
Here's a screenshot of what we see:
![console errors](https://d.pr/i/Ni7vii+)

Something to note is there is a .is-dark-theme class in the CSS.

#### Level of impact (Does it block purchases? Does it affect more than just one site?)
It's effecting a few sites so far I was able to reproduce on a test site and we've seen in two user cases:
27241243-hc
24527705-hc

### Temporary Workaround

Add caption via Media, https://wordpress.com/support/images/image-settings/#attachment-details

**Comment by carinapilar** [comment]
Got 2 users reporting this too:

28619507-hc
29970127-hc

Happens on Video blocks too, captions are also invisible. But only on specific themes, for the users I got, it was on Lodestar and Twenty Fourteen.

**Comment by fullofcaffeine** [comment]
Seems to be a Gutenberg core issue. Reported here: https://github.com/WordPress/gutenberg/issues/32470.

**Comment by formosattic** [comment]
4043511-zd-woothemes (on Twenty Twelve)
4062551-zd-woothemes (on Twenty Sixteen)
4063827-zd-woothemes (on Edda)

**Comment by carinapilar** [comment]
Another one with Canard: 4043777-zd-woothemes

**Comment by ericamQA** [comment]
A similar thing is happening on theme TwentyFifteen (at least) but for the "Add Title" field prompt which is also displaying as white text on a white background. 

![Add-Title-Is-White-On-White](https://user-images.githubusercontent.com/44411833/120896206-88074c00-c5ee-11eb-9955-131e2ec5ecf4.png)

I thought I'd add it here to help flesh out the scope of this issue and that it may not be limited to image captions. 

4043302-zen

Let me know if this is unrelated, technically speaking, and I'll make a new report for it.

**Comment by villanovachile** [comment]
Another report: 4044737-zen

**Comment by deancroyal** [comment]
WordPress.com forums: https://wordpress.com/forums/topic/edit-corrupts-my-existing-site/

Theme: Independent Publisher 2

Example of caption with white font.

<img width="400" alt="Screen Shot 2021-06-05 at 9 01 16 AM" src="https://user-images.githubusercontent.com/13387217/120897754-9f3d3e00-c5dc-11eb-99b0-5f5ec3716dea.png">

Once text is selected with mouse and cursor.

<img width="400" alt="Screen Shot 2021-06-05 at 8 58 19 AM" src="https://user-images.githubusercontent.com/13387217/120897682-48376900-c5dc-11eb-9f08-24ac2b5741e5.png">

**Comment by sanjeev00733** [comment]
Another report in 4044127-zen

**Comment by kathrynwp** [comment]
Another report in 4043773-zen

**Comment by louinel** [comment]
Another case here: 4047515-zen

**Comment by carinapilar** [comment]
Another 2 reports: 
- 4044574-zd-woothemes with Mistylook
- 4044649-zd-woothemes with Spatial

**Comment by formosattic** [comment]
18577569-hc

**Comment by umitkutlu1** [comment]
Kathryn ***@***.***> schrieb am So., 6. Juni 2021, 21:59:

**Comment by poojashetty18** [comment]
Another report in 4047164-zen

**Comment by fresatomica** [comment]
Another report here: https://wordpress.com/forums/topic/image-captions-12

**Comment by fstath** [comment]
Two more reports in this forum thread:

https://wordpress.com/forums/topic/captions-not-showing-up-and-cannot-add-new-captions-2/

**Comment by mrfoxtalbot** [comment]
This seems to be coming from the latest version of the Gutenberg plugins. I am closing this issue in favor of https://github.com/WordPress/gutenberg/issues/32470

Please feel free to add any additional details there.

Thank you all!

**Comment by KokkieH** [comment]
@mrfoxtalbot we shouldn't be posting in the Gutenberg repo with user reports, adding Zendesk and Happychat IDs that the community there don't have access to, so it might be good to keep this open in parallel with the GB issue for the purpose of user follow-up on our end once this is fixed.

On that note, some more reports from our forums: https://wordpress.com/forums/topic/i-can-not-add-captions-to-images-in-the-editor/
https://wordpress.com/forums/topic/photo-captions-10/
https://wordpress.com/forums/topic/write-caption-box/

**Comment by susanjanec** [comment]
Another report in 4047316-zen

**Comment by mrfoxtalbot** [comment]
I meant additional details about the issue itself, not user reports per se but thank you for clarifying that @KokkieH . 

Let's keep it open to make it easier for HEs to find it but we should really move the technical discussion over to the other issue.

Thanks again!

**Comment by adelineyaw** [comment]
Another two reports: 
4046610-zen
29885328-hc / 4049443-zen

**Comment by kspilarski** [comment]
Same case as 29673768-hc with Twenty Sixteen theme.

**Comment by vajrasar** [comment]
Another one with Ovation theme here: 4048659-zen

**Comment by kelasante** [comment]
Another report with retired Button theme here: 4047749-zen

**Comment by jinnypark** [comment]
Another report here: 40453980-zen with pub/twentysixteen theme.

**Comment by sajmes** [comment]
Also seeing this here 470432-hc with Dyad 2 theme. White text in editor but captions visible as expected when previewed/published.

**Comment by kathleenjross** [comment]
Seeing in 16393934-hc with the AltoFocus theme

**Comment by jinnypark** [comment]
Another report with pub/twentynineteen theme in 4046345-zen.

**Comment by jinnypark** [comment]
Another report with premium/obsidian in 4046329-zen.

**Comment by msilbers** [comment]
Reported in 28815890-hc using Twenty Eleven

**Comment by jinnypark** [comment]
Another one with pub/mistylook in 4044574-zen.

**Comment by syedshaon** [comment]
Another issue 4050304-zen using The Journalist v1.9

**Comment by syedshaon** [comment]
Another issue 4044884-zen using Twenty Ten

**Comment by Robertght** [comment]
A new report in 4047362-zen

**Comment by poojashetty18** [comment]
Another one on 4047978-zen

**Comment by SuzzieG** [comment]
Another one 4044655 - Zen

**Comment by paulostp** [comment]
4050622-zen Bloggy theme

**Comment by sanjeev00733** [comment]
3753906-hc
Theme: Twenty Eleven

**Comment by kspilarski** [comment]
4047752-zen
Obsidian theme

**Comment by sbathompson-he** [comment]
30034625-HC
Dara

**Comment by jp-imagines** [comment]
4050671-zen
Theme: Twenty Sixteen

**Comment by syedshaon** [comment]
Another issue 4048520-zen 
Theme: Twenty Sixteen

**Comment by retnonindya** [comment]
10608373-hc
Theme: Apostrophe 2

**Comment by devNigel** [comment]
Another report

17878446-hc
Theme: Twenty Eleven

**Comment by retnonindya** [comment]
1134659-hc
Theme: Twenty Sixteen

**Comment by pauljacobson** [comment]
We have another report in 12017961-hc with a follow-up in 4053927-zen

In this case, the missing captions are for Audio Blocks.

Theme: Apostrophe 2

**Comment by fstath** [comment]
Another case in the forums:
https://wordpress.com/forums/topic/cutlines-problem-disappearing-text/

Theme: Big Brother

**Comment by kdevnel** [comment]
Additional report: #21560276-hc
Follow-up: 4056034-zd-woothemes

**Comment by desnum** [comment]
A case at 10514393-hc

<img width="868" alt="white on white caption" src="https://user-images.githubusercontent.com/43796228/121376153-47ba0d80-c90f-11eb-9239-31a69e43325b.png">

**Comment by Sandkorner** [comment]
ZEN-4053563 

A user had the same issue with Audio Blocks. 

<img width="1248" alt="Screenshot 2021-06-09 at 15 53 05" src="https://user-images.githubusercontent.com/71094068/121395412-12d3a800-c95b-11eb-921a-52a6667c4076.png">

**Comment by dcoleonline** [comment]
#30054542-hc
Sundance theme

**Comment by jp-imagines** [comment]
30056233-hc
Theme: Adventure

**Comment by KirkwallDay** [comment]
29830898-hc
Theme: TextBook

**Comment by kaitohm** [comment]
User will need a response once resolved: 4054709-zd-woothemes

**Comment by KokkieH** [comment]
https://wordpress.com/forums/topic/missing-caption-option/
Theme: Twenty Ten

**Comment by poojashetty18** [comment]
30070019-hc
Theme: Twenty Sixteen

**Comment by dcoleonline** [comment]
#27704823-hc
Dyad 2

**Comment by fetenlakhal** [comment]
#4057430-hc
Radcliffe2

**Comment by atomicfan12** [comment]
#4056410-zen
Nikau

**Comment by sajmes** [comment]
4056391-zd-woothemes Theme: Dara

I could not replicate in Chrome when checking user's post.

**Comment by dcoleonline** [comment]
29062004-hc
Profile Theme

They couldn't see the captions under embedded YouTube videos.

**Comment by kaitohm** [comment]
Another case where the title area on pages is white, so a user wasn't able to find exactly where to add the title in the editor. This is the same as what @wpericam mentioned above on https://github.com/Automattic/wp-calypso/issues/53422#issuecomment-855253680 , but is happening on theme Photos

Report: 30074714-hc
Follow-up: 4060320-zd-woothemes

**Comment by jordesign** [comment]
Another case in #30079927-hc with Independent Publisher 2

**Comment by nagpai** [comment]
Another case with Ixion - #16258878-hc

**Comment by nagpai** [comment]
Looks like this issue was closed by @scruffian  on Core 
https://github.com/WordPress/gutenberg/issues/32470

Is this waiting to be merged here?

**Comment by nagpai** [comment]
Another one #16258878-hc
User on a retired theme though

**Comment by Robertght** [comment]
Another case in 4047764-zen

**Comment by davipontesblog** [comment]
Shifting priorities as there has been a surge of new reports in the last few days.

**Comment by davipontesblog** [comment]
This is being tracked here: https://github.com/WordPress/gutenberg/issues/32470
The fix does not seem to be applied to simple and atomic sites yet, but when it does, it should fix all of the reported incidences here.

**Comment by vajrasar** [comment]
Another case 4060987-zen

**Comment by KirkwallDay** [comment]
Another instance here: 5556905-hc
Theme: Obsidian

**Comment by xpurichan** [comment]
Reported in 29312177-hc
Theme: Twenty Fourteen

**Comment by rossanafmenezes** [comment]
Another report 4062520-zen
Theme: Independent Publisher 2

**Comment by rossanafmenezes** [comment]
Report: 4063393-zen
Theme: Karuna

**Comment by vajrasar** [comment]
Another case here: 4063616-zen

Themes: Ryu and Independent Publisher 2

**Comment by taggon** [comment]
This seems to be caused by the same bug with https://github.com/Automattic/wp-calypso/issues/53444

**Comment by AMcGrady** [comment]
Reported here: 22206970-hc

Theme: 
Ixion

**Comment by sophiegyo** [comment]
Noting another instance: 4064325-zen
Theme: Twenty Twelve

Edit 16 Jun 2021: Updated user to inform them of fix in #32477

**Comment by kriskarkoski** [comment]
4065088-zen
Theme: Twenty Eleven

**Comment by susanjanec** [comment]
Another report: 4064745-zen
Theme: Kubrick (retired)

**Comment by kathleenjross** [comment]
Another report: 4065124-zen
Theme: Manifest (retired)

**Comment by kathleenjross** [comment]
Another report: 4065307-zen
Theme: Nikau

**Comment by hacchism** [comment]
Another report: 4056410-zen
Theme: Nikau

**Comment by rinazrina** [comment]
Another one: 12001147-hc
Theme: Small Business

**Comment by fresatomica** [comment]
Another report: https://wordpress.com/forums/topic/general-20/
Theme: Mixfolio (retired)

**Comment by Robertght** [comment]
Another in 4066143-zen

**Comment by KokkieH** [comment]
https://wordpress.com/forums/topic/missing-caption-space-in-image-block/

**Comment by jackiejade** [comment]
4066177-zd-woothemes

**Comment by fresatomica** [comment]
Another report: https://wordpress.com/es/forums/topic/texto-debajo-de-imagenes/

**Comment by philnick206** [comment]
Another report #6034782-hc

**Comment by katiebethbrown** [comment]
Another report: 15442910-hc

**Comment by kathleenjross** [comment]
Another report: 30139843-hc
Theme: Canard

**Comment by sajmes** [comment]
Another report here: 30140831-hc

**Comment by joweber123** [comment]
Another report here 7061966-hc
Theme - Independent Publisher 2

**Comment by isocialtish** [comment]
Another report 4053012-hc
Sent them a follow up email: 4068783-zen

**Comment by inaikem** [comment]
Another report 21560276-hc
Follow up ticket here: 4056034-zen

**Comment by syedshaon** [comment]
Another issue 4061404-zen 
Theme: Nishita

**Comment by hacchism** [comment]
Another report in Japanese forum:
https://wordpress.com/ja/forums/topic/%E3%82%AD%E3%83%A3%E3%83%97%E3%82%B7%E3%83%A7%E3%83%B3%E3%81%8C%E8%A6%8B%E3%81%88%E3%81%AA%E3%81%84

**Comment by fresatomica** [comment]
Another one: https://wordpress.com/forums/topic/picture-caption/
Theme: Twenty Sixteen.

**Comment by mrfoxtalbot** [comment]
This has been fixed now. Closing!

**Comment by mrfoxtalbot** [comment]
@hacchism Could you please reply to the question the Japanse forums yourself? https://wordpress.com/ja/forums/topic/%E3%82%AD%E3%83%A3%E3%83%97%E3%82%B7%E3%83%A7%E3%83%B3%E3%81%8C%E8%A6%8B%E3%81%88%E3%81%AA%E3%81%84

Thank you!

**Comment by kaitohm** [comment]
@mrfoxtalbot - @hacchism is AFK, so [I've responded to the forum thread](https://wordpress.com/ja/forums/topic/%e3%82%ad%e3%83%a3%e3%83%97%e3%82%b7%e3%83%a7%e3%83%b3%e3%81%8c%e8%a6%8b%e3%81%88%e3%81%aa%e3%81%84/#post-23926) instead. We're all set there 👍🏼
